src/static/pycontw-2020/styles/page.scss

```
@@ -381,7 +381,16 @@ main {
     }
 }
 
+// customize privacy_policy and guidelines page ol
+ol.custom_ol {
+margin-top: 20px;
+}
 
+// customize privacy_policy and guidelines page h5 edited_info
+.edited_info{
+text-align: right;
+color: darkgray;
+}
```

please take care of ident & spacing. 
(p.s. consider adopting auto-formating tool, such as [Prettier](https://prettier.io/))
```suggestion
// customize privacy_policy and guidelines page ol
ol.custom_ol {
    margin-top: 20px;
}

// customize privacy_policy and guidelines page h5 edited_info
.edited_info {
    text-align: right;
    color: darkgray;
}
```